etc/config/sail.amazon.properties

```diff
@@ -0,0 +1,21 @@
+compilers=&sail
+
+defaultCompiler=sail_0_18
+
+group.sail.groupName=Sail
+group.sail.compilers=sail_0_18
+group.sail.isSemVer=true
+group.sail.baseName=sail
+
+group.sail.name=Sail
+group.sail.exe=sail
+group.sail.versionFlag=--version
+group.sail.options=-c
```

Comment:
Does every `sail` compilation require `-c` ? That would seem to be something that depends on what type of compilation is going on? Should maybe the `sail.ts` code add this (we do this in e.g. C++)

Reply:
Done